packages/docusaurus-types/src/index.js

```diff
@@ -1,10 +0,0 @@
-/**
- * Copyright (c) Facebook, Inc. and its affiliates.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- */
-
-/* eslint-disable */
-'use strict';
-Object.defineProperty(exports, '__esModule', {value: true});
```

Comment:
I have no idea what this does, any idea? :D

Reply:
This is how the compiled output of a `.ts` file with no value exports looks like. I guess it's just someone didn't know a package doesn't necessarily need a `main` entry?